mesh/meshdb.go

```diff
@@ -223,23 +208,22 @@ func (m *MeshDB) ForBlockInView(view map[types.BlockID]struct{}, layer types.Lay
 	return nil
 }
 
-func (m *MeshDB) layerBlockIds(index types.LayerID) ([]types.BlockID, error) {
-
-	ids, err := m.layers.Get(index.ToBytes())
+func (m *MeshDB) LayerBlockIds(index types.LayerID) ([]types.BlockID, error) {
```

Comment:
why did you change this to public? looks like no one is using it

Reply:
The `ninjaTortoise` uses it:
![image](https://user-images.githubusercontent.com/776358/72875237-83453f00-3cfc-11ea-90bb-fc7d17a6d71f.png)

Note that I didn't "change this to public". I just replaced a wrapper function that didn't seem to do anything with the actual function that the wrapper called. This is the deleted function:
https://github.com/spacemeshos/go-spacemesh/blob/66e9a7ead2d9b2f32e61cda2aad8b4ed5f4aedff/mesh/meshdb.go#L165-L178

Reply:
got it, very strange, i wonder how that happened...
maybe we had some issue with the slices returned from xdr ? (maybe just a bad merge)
anyway keep your fix if we run in to issues ill remember this change